### PR TITLE
Reinvent polka

### PR DESCRIPTION
**Pull request by evantravers**
Instead of prescribing a setup for everyone, we should move to a barebones skeleton that others can easily build off of. If someone wants a particular setup, they can fork off a fork of this barebones setup.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,35 +15,29 @@ Polka *will* nuke your current dotfiles, so first make sure to back up any macro
...
-* Git
-* Bash ~> 4.2
 
-## Setup Guides
+We recommend you have installed (through brew or whatever package manager you prefer) the following:
 
-* [Vim](http://blog.jerodsanto.net/2011/08/brew-install-vim/)
-* [Bash 4.*](http://stackoverflow.com/a/11704224)
-* [Pathogen](https://github.com/tpope/vim-pathogen)
+* bash
```
Comment (tmiller) [inline README.md]: We need to specify the version number for bash.

File: README.md [hunk 2]
```diff
@@ -13,35 +15,29 @@ Polka *will* nuke your current dotfiles, so first make sure to back up any macro
...
+    - It's going to create new .bashrc, .bash_profile, and .vimrc files (among others), as well as a new .vim folder and a ~/bin folder. The files will all actually live in your polka folder, and be symlinked to their proper places. This way you can keep them tied to the git repo, and update them with a simple git pull.
+    - BASH: If you check out your .bashrc, you're going to see something very different from what you are used to. Do not panic, it's the beginning of a bright new world for you. All your configurations, aliases, macros, and ruby loads now live in a bright and shiny new .bashrc.d folder. Each file is executable, linked back to the one in polka, and is numbered in order of execution. Feel free to add new files for your custom bindings, but leave the polka ones alone unless you intend to fork and you know what you are doing.
```
Comment (tmiller) [inline README.md]: This line needs to say that the file _must_ have the executable bit or else it won't load. Or we could change it to fit this statement.

File: README.md [hunk 3]
```diff
@@ -13,35 +15,29 @@ Polka *will* nuke your current dotfiles, so first make sure to back up any macro
...
+    - VIM: The fishbowl believes strongly in using vim as opposed to mvim, but feel free to modify and change at your leisure. If you aren't familiar with neobundle, take a look at your .vim folder. Instead of the usual menagerie, you are going to see a bundle folder, containing all the plugins we think are awesome. If you want to add your favorites to the list, fork the repo and add NeoBundle lines to it just like the ones in .vimrc, and you are done. All your plugins will be updated and kept happy and safe. Read [shougo's documentation](https://github.com/Shougo/neobundle.vim) for more details.
```
Comment (tmiller) [inline README.md]: s/fishbowl/authors/

File: README.md [hunk 4]
```diff
@@ -1,6 +1,8 @@
 # Polka
 
-Development configuration for the fishbowl. This is a very opinionated setup, strongly believing in mac osx, vim, ruby on rails, and the [base16](https://github.com/chriskempson/base16) color scheme.
+Development configuration for the fishbowl. This is a opinionated setup, strongly believing in linux, vim, and utilizes the [base16](https://github.com/chriskempson/base16) color scheme.
```
Comment (tmiller) [inline README.md]: s/Development configuration for the fishbowl./A development configuration setup a new computer quickly./

File: README.md (hunk 1 above)
Comment (evantravers) [inline README.md]: Just 4.*?

Comment (tmiller) [inline README.md]: bash >= 4.0

**Comment by evantravers** [comment]
Made changes based on @tmiller's helpful notes.

**Comment by tmiller** [comment]
Merged

**Pull request closed without merging**